index.js

```diff
@@ -5,11 +5,7 @@ exports.parse = function (str) {
 		return {};
 	}
 
-	if (str[0] === '&') {
-		str = str.substr(1, str.length);
-	}
-
-	str = str.trim().replace(/^(\?|#)/, '');
+	str = str.trim().replace(/^(\?|#|&)/, '');
```

Comment:
I doubt it has any practical performance difference, but it is indeed simpler. Thanks :)

https://github.com/sindresorhus/query-string/commit/f2833411bbe975d38b261ed46c96edde4bd0e6e7